### PR TITLE
✨ Add welcome Page as loading screen

### PR DESCRIPTION
**Pull request by DNaunin**
Forwards after three seconds

### CODE REVIEW

File: components/canvas/Drawing.module.css [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
   grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: 0.5fr 0.5fr 2fr 1fr;
+  grid-template-rows: repeat(3, 1fr);
   grid-column-gap: 1rem;
-  grid-row-gap: 2rem;
 }
 
-.patternimg {
-  grid-area: 1 / 1 / 3 / 2;
+.patternimg1 {
```
Comment (m-wiemers) [inline components/canvas/Drawing.module.css:8]: What is the difference between 1 and 2?
It is not usual to work with numbers here

File: components/canvas/Drawing.module.css [hunk 2]
```diff
@@ -16,28 +15,30 @@
...
-  grid-area: 3 / 1 / 4 / 3;
+.canvas1 {
+  grid-area: 2 / 1 / 3 / 3;
   border: 1px solid black;
   justify-self: center;
 }
 
-.canvasDesign {
-  grid-area: 3 / 1 / 4 / 3;
+.canvas2 {
```
Comment (m-wiemers) [inline components/canvas/Drawing.module.css:34]: same here ;-)

**Review by m-wiemers — APPROVED** [review]
gj 🚀

**Review by christophvogel — APPROVED** [review]
LGTM 👍

**Pull request merged**